src/main/resources/story/web_app/Trello.story

```diff
@@ -0,0 +1,37 @@
+Lifecycle:
+Examples:
+/tables/Trello.table
+
+Scenario: initialize variables
+Given I initialize STORY variable `invaliduser` with value `invaliduser`
+Given I initialize STORY variable `invalidpassword` with value `invalidpassword`
+Given I initialize STORY variable `invalidusermessage` with value `There isn't an account for this username`
+Given I initialize STORY variable `invalidpasswordmessage` with value `Incorrect email address and / or password. If you recently migrated your Trello account to an Atlassian account, you will need to use your Atlassian account password. Alternatively, you can get help`
+
+Scenario: successful login/ logout
+Given I am on a page with the URL 'https://trello.com/'
```

Comment:
Please add /properties/environment/trello/environment.properties file
Add next line there
web-application.main-page-url=https://trello.com/

It will let you use the step 
Given I am on the main application page

Reply:
It is not necessary for the training task, but its a best practice to keep main url in the properties. In real project you will be able to run the same tests on different envs (dev, qa, uat, prod) just setting the needed properties.

Reply:
Viktoryia, thank you for advices from real world! )